src/patternfly/components/Dropdown/dropdown.scss

```diff
@@ -303,13 +303,19 @@
   }
 }
 
+// remove this block at breaking change period
 .pf-c-dropdown__separator {
   height: var(--pf-c-dropdown__separator--Height);
   margin-top: var(--pf-c-dropdown__separator--MarginTop);
   margin-bottom: var(--pf-c-dropdown__separator--MarginBottom);
   background-color: var(--pf-c-dropdown__separator--BackgroundColor);
 }
 
+.pf-c-divider {
```

Comment:
With all of these changes for the divider in components, you've added `.pf-c-divider` to the global scope. You'll need to move the CSS blocks so they're scoped in the component they apply to.